node-operators/networks/collators/account-management.md

```diff
@@ -78,20 +77,32 @@ To map your author ID to your account, you need to be inside the [candidate pool
 
  1. Choose the account that you want to map your author ID to be associated with, from which you'll sign this transaction
  2. Select the **authorMapping** extrinsic
- 3. Set the method to **registerKeys()**
- 4. Enter the **authorId** (**NimbusId**). In this case, it was obtained via the RPC call `author_rotateKeys` in the previous section
- 5. For the **keys** field, enter the VRF key. This should have been obtained via an additional `author_rotateKeys` RPC call
- 6. Click on **Submit Transaction**
+ 3. Set the method to **setKeys()**
+ 4. Enter the **keys**. In this case, it is the response obtained via the RPC call `author_rotateKeys` in the previous section
+ 5. Click on **Submit Transaction**
 
 ![Author ID Mapping to Account Extrinsic](/images/node-operators/networks/collators/account-management/account-3.png)
 
 If the transaction is successful, you will see a confirmation notification on your screen. If not, make sure you've [joined the candidate pool](/node-operators/networks/collators/activities/#become-a-candidate){target=_blank}.
 
-![Author ID Mapping to Account Extrinsic Successful](/images/node-operators/networks/collators/account-management/account-4.png)
-
 ### Checking the Mappings {: #checking-the-mappings } 
 
-You can also check the current on-chain mappings by verifying the chain state. You'll need to click on **Developer** at the top of the page, then choose **Chain State** from the dropdown, and take the following steps:
+You can check the current on-chain mappings for a specific author ID or you can also check all of the mappings stored on-chain. For checking a specific author ID, you can take the first 64 hexadecimal characters of the concatenated public keys to get the author ID. To verify that the author ID is correct, you can run the following command with the first 64 characters passed into the `params` array:
+
+```
+curl http://127.0.0.1:9933 -H "Content-Type:application/json;charset=utf-8" -d   '{
+  "jsonrpc":"2.0",
+  "id":1,
+  "method":"author_hasKey",
+  "params": ["72c7ca7ef07941a3caeb520806576b52cb085f7577cc12cd36c2d64dbf73757a", "nmbs"]
```

Comment:
I find it confusing that here you have to slice the `0x` but in polkadot.js Apps you don't.

Reply:
well you pass in "nmbs" to make sure it's the author id, you can also run `author_hasKey` and pass in "rand" as a param with the last 64 characters... but it's not really applicable in this section since checking the mappings just requires you pass in the author id/nimbus key

i can create a separate section that explains it?